Kitodo/src/main/webapp/WEB-INF/resources/css/pattern-library.css

```diff
@@ -184,6 +184,53 @@ button .ui-icon {
     background: #0071bc;
 }
 
+.ui-chkbox .ui-chkbox-box, .ui-chkbox .ui-chkbox-box.ui-state-hover {
+    width: 40px;
+    height: 40px;
+    border-radius: 3px;
+    background: #999;
+    cursor: pointer;
+}
+
+.ui-chkbox .ui-chkbox-icon {
+  /*outline: 0;
```

Comment:
Why are you adding out commented style sheet information? If this is not needed, remove it please.